docs/APIRef.DetoxCLI.md

```diff
@@ -34,6 +34,53 @@ detox [options] [command]
 | --- | --- |
 | -h, --help | Output usage information |
 
+## Configuration Options
+
+Every flag can be passed in different ways, this section will show how `detox test --record-videos failing` can be expressed differently. Please note that you can use any combination of the usages below.
```

Comment:
I'd explain how the config is being prioritized (or did you already explain it and missed?)